### PR TITLE
py-pyppmd: update to 0.18.3

### PR DESCRIPTION
**Pull request by catap**
#### Description

It also update required dependencies and introduce a two new required ports.

Closes: https://trac.macports.org/ticket/65439

###### Type(s)

- [x] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 12.4 21F79 x86_64
Xcode 13.4.1 13F100

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: python/py-inflate64/Portfile [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                    port:py${python.version}-pytest
+
+    test.run        yes
+    test.cmd        py.test-${python.branch}
+    test.target
+    test.env        PYTHONPATH=${worksrcpath}/build/lib
+
+    pre-test {
+        test.env    PYTHONPATH=[glob -nocomplain ${worksrcpath}/build/lib*]
+    }
```
Comment (mascguy) [inline python/py-inflate64/Portfile:39]: Are all of the test-related items necessary, apart from `test.run yes`?

File: python/py-pybcj/Portfile [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    test.run        yes
+    # most test=suites are run using "pytest", if not then the lineb elow should
+    # be modified (e.g., "nosetests-${python.branch}" or "${python.bin} -m unittest"
+    test.cmd        py.test-${python.branch}
+    test.target
+    test.env        PYTHONPATH=${worksrcpath}/build/lib
+
+    pre-test {
+        test.env    PYTHONPATH=[glob -nocomplain ${worksrcpath}/build/lib*]
+    }
```
Comment (mascguy) [inline python/py-pybcj/Portfile:45]: Same question as before.

**Comment by catap** [comment]
Christopher, indeed. Fixed

**Review by mascguy — APPROVED** [review]
LGTM. @kurthindenburg ?

**Comment by catap** [comment]
@mascguy same question here: which is timeout for response from maintainer?

**Comment by catap** [comment]
> We need to wait at least 72 hours, so one more day. :-)

See you tomorrow when!

**Comment by mascguy** [comment]
Maintainer timeout exceeded, merging.

**Pull request merged**